view/Requirements.php

```diff
@@ -1005,6 +1005,16 @@ protected function path_for_file($fileOrUrl) {
 		}
 	}
 
+	protected function generateNonce($filePath) {
+		if(!file_exists($filePath)) return 'notfound';
+
+		if(Config::inst()->get('Requirements_Backend', 'nonce_format') === 'md5') {
```

Comment:
The config which turns this on is `suffix_requirements`. Should we use the same naming scheme for the format? E.g. `suffix_format`?